### PR TITLE
feat(core): Add support for InCallHandRaise proto message

### PR DESCRIPTION
**Pull request by thisisamir98**
## Description

Updated core logic to handle the new message type.

Implements https://github.com/wireapp/generic-message-proto/pull/90

### CODE REVIEW

File: packages/core/src/conversation/InCallHandRaiseType.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/**
+ * true if the hand is raised, false if lowered
+ */
+export type InCallHandRaiseType = boolean;
```
Comment (e-maad) [inline packages/core/src/conversation/InCallHandRaiseType.ts:23]: One file for one type 😟 
Do you think we should move all the types in the same file?

Comment (e-maad) [reply packages/core/src/conversation/InCallHandRaiseType.ts:23]: I think we don't need it we can directly use boolean.

File: packages/core/src/conversation/content/InCallHandRaiseContent.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {InCallHandRaiseType} from '../InCallHandRaiseType';
+
+export interface InCallHandRaiseContent {
+  is_hand_up: InCallHandRaiseType;
```
Comment (e-maad) [inline packages/core/src/conversation/content/InCallHandRaiseContent.ts:23]: Out of curiosity, why we are not following variable naming convention here?
```suggestion
  isHandUp: boolean;
```

**Review by e-maad — COMMENTED** [review]
Added few suggestions.

File: packages/core/src/conversation/content/InCallHandRaiseContent.ts (hunk 2 above)
Comment (thisisamir98) [reply packages/core/src/conversation/content/InCallHandRaiseContent.ts:23]: this is because of the messaging proto

Comment (thisisamir98) [reply packages/core/src/conversation/content/InCallHandRaiseContent.ts]: done in https://github.com/wireapp/wire-web-packages/pull/6636/commits/59db7920db7ee8d294af0895a52eefc607f0fbd7

File: packages/core/src/conversation/InCallHandRaiseType.ts (hunk 1 above)
Comment (thisisamir98) [reply packages/core/src/conversation/InCallHandRaiseType.ts]: done in https://github.com/wireapp/wire-web-packages/pull/6636/commits/59db7920db7ee8d294af0895a52eefc607f0fbd7

File: packages/core/src/conversation/message/MessageBuilder.ts [hunk 3]
```diff
@@ -309,6 +311,20 @@ export function buildInCallEmojiMessage(payloadBundle: InCallEmojiMessage['conte
   });
 }
 
+export function buildInCallHandRaiseMessage(payloadBundle: InCallHandRaiseMessage['content']): GenericMessage {
+  const {isHandUp} = payloadBundle;
+
+  const handRaise = InCallHandRaise.create({
+    isHandUp: isHandUp,
```
Comment (e-maad) [inline packages/core/src/conversation/message/MessageBuilder.ts:318]:
```suggestion
    isHandUp,
```

**Review by e-maad — APPROVED** [review]
LGTM 🚀

**Pull request merged**